src/libs/ajax.js

```diff
@@ -904,6 +903,11 @@ const DataRepo = signal => ({
         return res.json()
       }
     }
+  },
+
+  requestAccess: async id => {
+    const res = await fetchRawls(`dunno/what/this/is/${id}/requestAccess`, _.merge(authOpts(), { signal }))
```

Comment:
Do we have an endpoint for this yet?

Reply:
Not yet, that should be coming in the future!

Reply:
Consider adding a TODO to call this out.